feature-pack/src/main/resources/modules/system/layers/base/org/infinispan/hibernate-cache/main/module.xml

```diff
@@ -43,6 +51,7 @@
         <module name="org.hibernate.jipijapa-hibernate5" services="import"/>
         <module name="org.infinispan" services="import"/>
         <module name="org.infinispan.commons"/>
+        <module name="org.infinispan.hibernate-cache-commons" services="import"/>
```

Comment:
Why was this dependency added? It's also listed in the resources so it will be copied into this module.

Reply:
hmm, yeah, weird for it to be a resource and self referenced.  should be no reason for that, must of missed it.

Reply:
removed.